interoperability-layer-openhim/swarm.sh

```diff
@@ -53,6 +53,9 @@ function initialize_package() {
 
     if [[ "${CLUSTERED_MODE}" == "true" ]] && [[ "${ACTION}" == "init" ]]; then
       try "${COMPOSE_FILE_PATH}/initiate-replica-set.sh $STACK" throw "Fatal: Initiate Mongo replica set failed"
+    else
```

Comment:
Instead of hardcoding the 1 here in `mongo-1`, can we rather run a `docker node inspect self` to see which number should be used? This will ensure it can run on all nodes, instead of only the leader node. This logic should also be updated in the [initiate-replica-set.sh#LL54C46](https://github.com/jembi/platform/blob/eeed245991a1e927bb178fca7a7e372da5aeb41a/interoperability-layer-openhim/initiate-replica-set.sh#LL54C46-L54C46) script. Both should default to 1 if the node inspect didn't return a value.

Reply:
sorted in https://github.com/jembi/platform/pull/238/commits/bf613541f129f63f258c5f2bdb2c41e3c98d35c9